src/main/resources/standardTestCases/monitoring.test

```diff
@@ -1,15 +1,16 @@
-//Currently doesn't work in 0.20
+#Currently doesn't work in 0.20, monitoring log is empty and
```

Comment:
Andddd sorry Chris, nothing of actual excitement was discovered.....I thought I discovered that the monitoring log file being sent to the wrong bucket, but that was due to a change in my Cromwell conf 😌